tests/modules/nf-core/kofamscan/main.nf

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNTAR     } from '../../../../modules/nf-core/untar/main.nf'
+include { GUNZIP    } from '../../../../modules/nf-core/gunzip/main.nf'
+include { KOFAMSCAN } from '../../../../modules/nf-core/kofamscan/main.nf'
+
+workflow test_kofamscan {
+
+    fasta = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+    profiles = [ [], file(params.test_data['sarscov2']['genome']['kofamscan_profiles_tar_gz'], checkIfExists: true) ]
+    ko_list = [ [], file(params.test_data['sarscov2']['genome']['kofamscan_ko_list_gz'], checkIfExists: true) ]
+    format = 'detail'
+
+    UNTAR( profiles )
+    GUNZIP( ko_list )
+    KOFAMSCAN ( [ [id:'test'], fasta ], UNTAR.out.untar.map{ it[1] }, GUNZIP.out.gunzip.map{ it[1] }, format )
+}
```

Comment:
@nvnieuwk Since ``format`` val input is removed, how do you suggest passing ``ext.args`` in the test (``--format detail-tsv``)? Do you know any other module I could use as example? Thanks!

Reply:
You can supply this inside `tests/modules/nf-core/kofamscan/nextflow.config` like this:
```
    withName: KOFAMSCAN {
        ext.prefix = { "${meta.id}.kofamscan" }
        ext.args = "--format detail-tsv"
    }
```

Your code will check if this format definition is inside of `args` and define the format from there automatically :)

Reply:
You'll probably want to use an alias though so your other test doesn't also output a TSV file

Reply:
So @nvnieuwk, if I understand well, there should be then two nextflow.config files (```nextflow.config``` and ```nextflow.tsv.config``` ) in test directory?

Reply:
All configuration should be in `nextflow.config` since other configs in the folder won't be used by `pytest`. I'll look for an example :)

Reply:
`nextflow.config`: https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/bcftools/convert/nextflow.config
`test_workflow` : https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/bcftools/convert/main.nf

Reply:
Feel free to ask if something is not clear :)

Reply:
This is an even better example: https://github.com/nf-core/modules/tree/master/tests/modules/nf-core/gatk4/collectreadcounts

This module also uses `--format`